pkgbuild/pkgbuild.sh

```diff
@@ -80,16 +80,24 @@ case $JVM in
   openj9)
     IDENTIFIER="net.adoptopenjdk.${MAJOR_VERSION}-openj9.${TYPE}"
     DIRECTORY="adoptopenjdk-${MAJOR_VERSION}-openj9.${TYPE}"
-    /usr/libexec/PlistBuddy -c "Set :CFBundleGetInfoString AdoptOpenJDK (OpenJ9) ${FULL_VERSION}" "${INPUT_DIRECTORY}/Contents/Info.plist"
-    /usr/libexec/PlistBuddy -c "Set :CFBundleName AdoptOpenJDK (OpenJ9) ${MAJOR_VERSION}" "${INPUT_DIRECTORY}/Contents/Info.plist"
+    BUNDLE="AdoptOpenJDK (OpenJ9)"
```

Comment:
Is this line needed if the case below actually does the setting?  I notice you don't follow the same pattern for Hotspot...

Reply:
It allows for more variants to be added if needed